src/components/summary/_macro.spec.js

```diff
@@ -303,12 +303,26 @@ const EXAMPLE_SUMMARY_MULTIPLE_GROUPS = {
   ],
 };
 
+// To address a DAC issue, we've disabled specific axe definition list rules causing test failures.
+// While resolving it would require a significant refactor, the failures are deemed non-critical for accessibility,
+// leading to their removal in this context. [https://github.com/ONSdigital/design-system/issues/3027]
```

Comment:
Can you add something to outline and explain this change in the PR description too